ConsoleApp2/ConsoleApp2/ListExercises.cs

```diff
@@ -50,19 +50,27 @@ public void Exercise()
 
 
             Console.WriteLine(refTypes.Count());
-            if (refTypes.Contains(adam))
-            {
-                adam.Name = String.Empty;
-            }
-            var copy = refTypes;
-            Console.WriteLine(refTypes.Count());
-            for (int i = 0; i < refTypes.Count; i++)
+            //if (refTypes.Contains(adam))
+            //{
+            //    adam.Name = String.Empty;
+            //}
+            
+            var copy = refTypes.Select(person => new Person { Name = person.Name }).ToList();
```

Comment:
try this
var copy = refTypes.ToList();
is the output the same?

Reply:
Yes output will be same